### ISSUE TITLE
[Bug] Bulk Action on cases does not work

### ISSUE THREAD

**Issue by Tyrell20** [TheHive-Project/TheHive#1871]
### Request Type
Bug

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | RedHat 7.9
| TheHive version / git hash   | 4.1.0-1
| Package Type              | RPM
| Browser type & version    | Google Chrome Version 89.0.4389.90

### Problem Description
Use bulk action, after the selection of several case, does not propagate the action for all case. TheHive shows and requests to manage cases individually

### Steps to Reproduce
1. Select more than one case;
2. Click on "number of case selected" to perform a bulk action;
3. Select close case.

### Possible Solutions
The "Bulk Action" should be propagate the information about cases selected for all of them avoiding to repeat the information individually showing the related info of closure.

**Comment by jeromeleonard** [comment]
Yes, for each case, you need to specify a custom status and a summary. This is not a bug. Maybe it could be improved.

**Comment by Tyrell20** [comment]
Many thanks @jeromeleonard. Exactly it could be useful to manage several similar case in one time.

**Comment by Litie78** [comment]
From my point of view, an idea of improvement, could be let the user decide when close all cases at the same time with the same Comment, maybe via a checkbox.

**Comment by nadouani** [comment]
Well, this feature is mostly made for those who want to close a decent number of cases, and need to specify the details of each closure.

If you want to close 100 cases but a simple click, just script it.

**Comment by diogodaz** [comment]
does anyone have this script to share?
In my case I need to close ALL Waiting task at once.
